content/millennium/r4/care-provision/care-plan.md

```diff
@@ -0,0 +1,146 @@
+* [Encounter](http://hl7.org/fhir/r4/careplan-definitions.html#CarePlan.encounter){:target="_blank"}
+* [Effective Period](http://hl7.org/fhir/r4/careplan-definitions.html#CarePlan.period){:target="_blank"}
+* [Author](http://hl7.org/fhir/r4/careplan-definitions.html#CarePlan.author){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:care_plan, :r4) %>
+
+
+## Search
+
+Search for CarePlans that meet supplied query parameters:
+
+    GET /CarePlan?:parameters
+
+_Implementation Notes_
+
+* The [CarePlan.activity] element is not supported and will not be returned. This means that the [CarePlan.activity.detail.status] and [CarePlan.activity.detail.prohibited] modifier elements will not be returned.
+
+* The description of the CarePlan is included in the [CarePlan.text.div] element as ASCII text.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: true, system: true) %>
+
+### Parameters
+
+ Name         | Required? | Type          | Description
+--------------|-----------|---------------|--------------
+ `_id`        | See notes | [`token`]     | The logical resource id associated with the resource.
+ `date`       | N         | [`date`]      | Time period plan covers. Example: `date=ge2016&date=le2017 (Jan 1, 2016 - Dec 31, 2017)` or `date=ge2016-08-24T12:00:00.000Z&date=le2017-01-24T12:00:00.000Z`
+ `patient`    | See notes | [`reference`] | Who care plan is for. Example: `patient=1316024`
+ `category`   | See notes | [`token`]     | Only support 'assess-plan'
+ [`_count`]   | N         | [`number`]    | Number of results per page.
+
+
+Notes:
+
+- The `_id` parameter
+  - May not be provided with any other parameters.
+  - When provided, `_count` is ignored.
+
+- The `date` parameter
+  - When provided, must use both `ge` and `le` prefixes in the same search 
+    - The lower value must have the `ge` prefix and the higher value must have the `le` prefix
+    - If date precision must be consistent
+  - May be combined with the patient and category parameters.
+
+- The `patient` parameter
+  - This is required when `_id` is not provided.
+
+- The `category` parameter
+  - Must be provided when `_id` is not provided.
+  - Only supports the code `assess-plan`.
+  - This can be combined with the patient parameter.
+
+### Headers
+
+ <%= headers %>
+
+### Example retrieve care plan
+
+#### Request
+
+    GET https://fhir-open.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/CarePlan?patient=1316020&category=assess-plan
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_care_plan_bundle) %>
+
+<%= disclaimer %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## Retrieve by id
+
+List an individual CarePlan by its id:
+
+    GET /CarePlan/:id
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: true, system: true) %>
```

Comment:
Can we update this to reflect that patient authorization is currently not supported.

Reply:
Removed patient authorization